### ISSUE TITLE
Feedback - Module 0. Setting up

### ISSUE THREAD

**Issue by helendduncan** [alan-turing-institute/rse-course#135]
Things that came up in the 2022 set-up clinic for the RSE course.

Terminal cannot find the (installed) conda. 
Solution: Edit .zshrc to include 
```bash
export PATH="/path/to/users/conda/or/anaconda/install/anaconda3/bin:$PATH"
```

Query regarding C++ compiler for Windows
In the [installation instructions](https://alan-turing-institute.github.io/rse-course/html/course_prerequisites/04_jupyter.html) there is a section about installing C++ for Windows (presumably for module 9?). The "see here for details" link is to a non-ATI repo with some help but it appears that a `setup.py` is needed. This isn't in the repo that I could see.

**Comment by jemrobinson** [comment]
I think [this](https://github.com/cython/cython/wiki/CythonExtensionsOnWindows#using-windows-sdk-cc-compiler-works-for-all-python-versions) might link to the correct part of that page. What do you think @helendduncan ?

**Comment by helendduncan** [comment]
Yes - we did end up looking there but I was unsure as under "Less useful information" was a warning not to use MinGW-w64 which I believe the student had. The student had installed VS code and they got that up and running - but it was a bit opaque (to me at least) about what to do for this on a Windows machine (and when it would be needed)?

**Comment by jemrobinson** [comment]
I think the basic point is:

- you need to compile Python for Windows
- any Cython extensions need to use the same compiler that you used to compile Python

If you are using Anaconda, then the compiler is `MSC v.1915 64 bit (AMD64)` [according to this StackOverflow post](https://stackoverflow.com/questions/52864588/how-to-install-cython-an-anaconda-64-bits-with-windows-10).
